code/game/objects/items/ego_weapons/he.dm

```diff
@@ -91,6 +92,25 @@
 	attribute_requirements = list(
 							FORTITUDE_ATTRIBUTE = 40
 							)
+	var/living = FALSE
+	var/rage = FALSE
+
+/obj/item/ego_weapon/fury/attack(mob/living/target, mob/living/carbon/human/user)
+	if(!CanUseEgo(user))
+		return
+	if(target.stat != DEAD)
+		living = TRUE
+	..()
+	if(force != initial(force))
+		force = initial(force)
+
+	if(target.stat == DEAD && living)
+		if(!rage)
+			to_chat(user, "<span class='userdanger'>LONG LIVE THE QUEEN!</span>")
+			rage = FALSE
+		force *= 3
+		rage = TRUE
+		living = FALSE
```

Comment:
This is a very weird way to do this.
Just shove the rage check above, before damage, just like it's done with justitia and every other weapon using combo "system".

Reply:
the "Rage" check is JUST for the message so you don't get a million messages in chat.

Reply:
Also, it gives you bonus damage for your NEXT attack if you kill something. Not this one.